src/service/auth/asyncStorage.tsx

```
@@ -0,0 +1,83 @@
+    const token = await localforage.getItem("userToken");
+    const refreshToken = await localforage.getItem("refreshToken");
+    if (token !== null && refreshToken !== null) {
+      return { token, refreshToken };
+    } else {
+      return null; // Return null if no token found
+    }
+  } catch (error) {
+    console.error("Error retrieving token:", error);
+    throw error;
+  }
+  return null;
```

Biome

[error] This code is unreachable

... because either this statement will return from the function, ...

... this statement will return from the function, ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)